spec/models/comment_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'spec_helper'
+
+describe Comment do
+  context "Validate" do
+    it 'should have a valid factory' do
+      FactoryGirl.create(:comment)
+    end
+
+    it "Comment must belongs to article" do
+      comment = FactoryGirl.create(:comment)
+
+      expect(comment.respond_to?(:article)).to be true
+    end
+
+    it "Should pass create comment" do
+      expect do
+        comment = FactoryGirl.create(:comment)
```

Comment:
@phendych this is not necessary, you don't need to test against ActiveRecord's create method. you can assume this always works, the Rails guys will have a test for this, this isn't in your scope.